intera_interface/src/intera_motion_interface/motion_waypoint_options.py

```diff
@@ -57,6 +57,19 @@ def get_accel_preset(accel_name):
         rospy.logwarn('Did not recognize accel preset. Skipping.')
         return None
 
+    @staticmethod
```

Comment:
This whole file is very Sawyer-specific, which I'm not a huge fan of, but likely we're not going to fix it this pull request. Just throwing that out there to noodle on for the future - I'd like to see a configuration file where all the sawyer presets and magic numbers live.